.github/workflows/ci.yml

```diff
@@ -29,4 +29,4 @@ jobs:
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
       - name: "Run unit & integration tests"
-        run: mvn -V -ntp verify --file pom.xml
+        run: mvn -U -V -ntp verify --file pom.xml
```

Comment:
Strictly speaking, we shouldn't need to use the `-U` arg, as we shouldn't be SNAPSHOT versions for the parent POM or dependencies. Doesn't hurt to have, but may make the maven cache in this GH action a bit less useful